resources/example_doip.lua

```diff
@@ -0,0 +1,10 @@
+Main = {
+	ANNOUNCE_NUM = 3,
+	ANNOUNCE_INT = 500,			-- in milliseconds
```

Comment:
Abbreviation INT could be hard to understand.
Use full word instead ANNOUNCE_INTERVAL.

Reply:
You are right, that is not a meaningful naming